README.md

```diff
@@ -28,6 +28,10 @@ Our process is as follows:
     already many cases where switching from `unsafe` to safe alternateives has
     _increased_ performance, so simply guessing that performance will regress
     is not enough.
+  * If the `unsafe` block is sound, but can be converted to safe code without
```

Comment:
* is sound, **but** can

Then put this bullet point first in the list so that we're listing the best outcome first, then progressively worse outcomes.